lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2109,7 +2109,23 @@ Value *SPIRVToLLVM::transValueWithoutDecoration(SPIRVValue *BV, Function *F,
   }
   case OpCopyLogical: {
     SPIRVCopyLogical *CL = static_cast<SPIRVCopyLogical *>(BV);
-    return mapValue(BV, transSPIRVBuiltinFromInst(CL, BB));
+
+    auto *SrcTy = transType(CL->getOperand()->getType());
+    auto *SrcAI = new AllocaInst(SrcTy, 0, "", BB);
```

Comment:
Since you have an IRBuilder below, you might as well hoist that up and use its `CreateAlloca`, `CreateStore` and `CreateLoad` member functions; that will save you from passing the `"", BB` arguments all the time.

Reply:
Done!